src/Asset/Asset.js

```diff
@@ -31,18 +31,6 @@ export class Asset {
     return assets[0]
   }
 
-  findByIds(ids) {
```

Comment:
Why we are removing it?

Reply:
Not used anywhere and was a security concern, ids was being interpolated and could be SQL injected